docs/roman_datamodels/datamodels/models.rst

```diff
@@ -10,12 +10,12 @@ format.
 
 Each model instance is created to contain a variety of attributes and data that
 are needed for analysis or to propagate information about the file and the
-contents of the file. For example, the `ImageModel` class has the following
-arrays associated with it:
+contents of the file. For example, the ``ImageModel`` class
```

Comment:
Why don't we add API documentation so that there is an actual link to be populated here.

Reply:
I tried `roman_datamodels.datamodels.ImageModel` and `datamodels.ImageModel` and got
```
/Users/zburnett/projects/roman_datamodels/docs/roman_datamodels/datamodels/models.rst:11: WARNING: py:obj reference target not found: roman_datamodels.datamodels.ImageModel
```

Reply:
Your getting that because there is no API documentation that got built. So there is nothing to link to. My suggestion was to have sphinx auto create/build the api documentation.